pages/legacy/collections_confirm_remove.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.legacy.base import PrivatePage
+
+from selenium.webdriver.common.by import By
+
+
+class CollectionsConfirmRemove(PrivatePage):
+
+    _form_locator = (By.CSS_SELECTOR, 'form[action="folder_delete_confirm"]')
+    _remove_button_locator = (By.CSS_SELECTOR, 'input[type="submit"][name="form.button.delete"]')
+
+    @property
+    def form(self):
+        return self.find_element(*self._form_locator)
+
+    @property
+    def remove_button(self):
+        return self.form.find_element(*self._remove_button_locator)
+
+    def confirm(self):
```

Comment:
Should confirm be clicking the remove button?

Reply:
not really sure. I think @Dantemss wrote this part and I modified it a bit for my own object. They share some similar features